src/main.js

```diff
@@ -3,17 +3,47 @@
 // App configuration
 const config = require("./config");
 
+// MongoDB setup
+const mongoose = require("mongoose");
+mongoose.connect(config.mongoURI);
+
 // Express setup
 const express = require("express");
 const app = express();
 
 app.use(express.static(config.staticDir));
 
+//Include Room module
+var room = require("./room"); 
+
 // Endpoints
 app.get("/api/test", (req, res) => {
 	res.send({status: "ok"});
 });
 
+//Creating room endpoint
+app.get("/api/creroom", (req, res) => {
```

Comment:
Make a POST request handler (`app.post("...", (req, res) => { ... })` instead of `app.get`.  

Change endpoint url to `/api/rooms`.